apps/www/components/Profile/EditProfile/Submit.js

```diff
@@ -27,6 +27,24 @@ const Submit = ({ me, user, t, state, setState, update }) => {
     return null
   }
 
+  // Function to ensure URL has protocol and strip parameters
+  const normalizeUrl = (url) => {
+    // Add protocol if missing
+    const urlWithProtocol = url.indexOf('https://') === -1 && 
+      url.indexOf('http://') === -1
+      ? 'https://' + url
+      : url
+    
+    // Strip parameters
+    try {
+      const urlObj = new URL(urlWithProtocol)
+      return `${urlObj.protocol}//${urlObj.host}${urlObj.pathname}`
```

Comment:
I feel like the purpose of this would be more transparent by setting the search/query params to '' and converting the url to string, but it's a detail.